1-js/02-first-steps/15-function-basics/4-pow/task.md

```diff
@@ -4,16 +4,16 @@ importance: 4
 
 # Function pow(x,n)
 
-Write a function `pow(x,n)` that returns `x` in power `n`. Or, in other words, multiplies `x` by itself `n` times and returns the result.
+एक function लिखें `pow(x,n)` जो `x` को `n` के शक्ति में लौटाता है। या, दूसरे शब्दों में, `x` को अपने आप से `n` गुणा से गुणा करता है और परिणाम देता है।
```

Comment:
This line also needs to be re-looked. Let's try to use proper mathematical terms in Hindi

Reply:
done